app/assets/javascripts/short-form/components/preferenceCheckbox.js.coffee

```diff
@@ -9,16 +9,20 @@ angular.module('dahlia.components')
     translatedDescription: '@'
     customDescription: '@'
     shortDescription: '@'
+    readMoreUrl: '@'
     onChange: '&'
   templateUrl: 'short-form/components/preference-checkbox.html'
 
   controller: ['ListingDataService', 'ListingPreferenceService', (ListingDataService, ListingPreferenceService) ->
     ctrl = @
 
-    @showDescription = false
     listingPreference = ListingPreferenceService.getPreference(@preference, ListingDataService.listing)
     if listingPreference
       @moreInfoLink = listingPreference.readMoreUrl
+    else
```

Comment:
nit: On a component API level I kind of feel like this should be flipped, like if you pass an explicit read more link it should override the salesforce version

like 
```
if !!@readMoreUrl
  @moreInfoLink = @readMoreUrl
else if listingPreference
  @moreInfoLink = listingPreference.readMoreUrl
```

just nitpicking though since I know this is hopefully not a component we'll have to use much more.

Reply:
updated, should probably QA this again

Reply:
actually, i wonder if I still need this w/ the change I made back to use the listing preference ID instead of calling the fields five88. I'll investigate now

Reply:
OK I was able to use getPreferenceById() to simplify this a bit. I did a bit of QA but would appreciate another set of eyes too!